brownie-config.yml

```diff
@@ -1,7 +1,7 @@
 # use Ganache's forked mainnet mode as the default network
 # NOTE: You don't *have* to do this, but it is often helpful for testing
 networks:
-  default: mainnet-fork
+  default: development
```

Comment:
i don't think this change is needed?

Reply:
I thought I was going to unit test GasBenefactor in this repo, and `yearn-vault` has `development` as default and specifies `mainnet-fork` in CI when testing integration testing. 
But since I won't be testing GasBenefactor needed, several changes might be rolled back. Good catch.